addon/ActivityStreams.js

```diff
@@ -105,12 +104,8 @@ function ActivityStreams(metadataStore, options = {}) {
   );
   this._experimentProvider.init();
 
-  this._tabTracker = new TabTracker(
-    this.appURLs,
-    options.clientID,
-    this._memoized,
-    this._experimentProvider.experimentId
-  );
+  this._tabTracker = tabTracker;
+  this._tabTracker.init(this.appURLs, this._memoized, this._experimentProvider.experimentId);
```

Comment:
We will need TabTracker to operate independently of the ActivityStream object for it to track metrics for the tiles cohort of the SHIELD study, which means it will need to be able to operate without ActivityStream being enabled.  If any of these fields are necessary for TabTracker to send metrics payloads we should move them out as well.